package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@openst/mosaic-chains",
-  "version": "0.1.0-alpha.1",
+  "version": "0.1.0-alpha.2",
```

Comment:
```suggestion
  "version": "0.1.0-alpha.5",
```

Reply:
also please change package version in package.json to `0.1.0-alpha.5`

Reply:
Done!